R/collapse_intron_counts.R

```diff
@@ -19,13 +19,14 @@
 #'
 #' # include unspliced cDNA in final counts matrix
 #' collapse_intron_counts(counts,
-#'                        which_counts = "unspliced")
+#'                        which_counts = "total")
 #' }
 #'
 #'
 #'
 collapse_intron_counts <- function(counts,
-                                   which_counts = c("spliced", "unspliced")){
+                                   which_counts = c("spliced", "unspliced", "total")){
+
   which_counts <- match.arg(which_counts)
 
```

Comment:
I feel like we should warn people about the fact that "unspliced" not a good choice. 
I don't love my wording here, but I want something...

```suggestion
  which_counts <- match.arg(which_counts)

  if (which_counts ==  "unspliced"){
    warn("which_counts = 'unspliced' is deprecated, as its behavior is ambiguous (and included both spliced and unspliced counts) in previous versions"
```